docs/command-line-slangc.md

```diff
@@ -85,6 +85,7 @@ For completeness, here are the options that `slangc` currently accepts:
 * `-pass-through <name>`: Don't actually perform Slang parsing/checking/etc. on the input and instead pass it through more or less modified to the existing compiler `<name>`"
   * `fxc`: Use the `D3DCompile` API as exposed by `d3dcompiler_47.dll`
   * `glslang`: Use Slang's internal version of `glslang` as exposed by `slang-glslang.dll`
+  * 'dxc': Use DirectXShaderCompiler (https://github.com/Microsoft/DirectXShaderCompiler)
```

Comment:
Eventually we could get all cutesy and use proper markdown link syntax here, so that this renders as just a link. IIRC the markdown should be:

```
[DirectXShaderCompiler](https://github.com/Microsoft/DirectXShaderCompiler)
```

which should render as:

> [DirectXShaderCompiler](https://github.com/Microsoft/DirectXShaderCompiler)